tests/tasks/lib/test_tasks.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+import os
+import importlib
+from collections import Counter
+import numpy as np
+
+from jiant.shared import model_resolution
+from jiant.tasks import create_task_from_config_path
+from jiant.utils.testing.tokenizer import SimpleSpaceTokenizer
+
+CLASSIFICATION_TASKS = ['mnli', 'sst']
```

Comment:
Looking for a way to abstract this out further (maybe task config) to reduce the parts of code needing to be touched when adding a task.

Reply:
Could you clarify? Also, are we intending to put all task tests into one file? See above comment: I think we should not cluster beyond tasks of the same archetype, and tasks that are fairly unique should have their own files.

Reply:
Completely agree that we should not put task test into one file. This will not be the final structure for all task tests. I am trying to figure out the one source of truth to rule them all for task type. Then, I would grab all classification tasks and run this test on them. Not sure what the best way to do this is yet...